src/Engine/UnitActionLifecycles/LoadLifecycle.java

```diff
@@ -168,6 +168,7 @@ public void performEvent(MapMaster gameMap)
         gameMap.removeUnit(unitCargo);
         unitCargo.x = -1;
         unitCargo.y = -1;
+        unitTransport.heldUnits.remove(unitCargo); // Make sure we can't double-load
```

Comment:
Why is this needed? Shouldn't the unload happen as part of the `LAUNCH` event?
I'm still not even sure why we allow launching back to the launching ship.

Reply:
Yes, actually. However, the `LAUNCH` assumes that moves ending on its space are *not* followed by a `LOAD`, so it queues another `LOAD` at the end.
The double `LOAD` is why you ended up seeing two spaces taken up. This fixes that, though most paths for that to happen are now closed off (You could have an AI manually construct a `LAUNCH` like that, but you can't do it in the UI and `LaunchFactory` isn't gonna do it for you)